src/pages/patientView/PatientViewGenePanelModal/PatientViewGenePanelModal.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { GenePanel } from 'shared/api/generated/CBioPortalAPI';
+import { observer } from 'mobx-react';
+import GenePanelModal from 'shared/components/GenePanelModal/GenePanelModal';
+import GenesList from './GenesList';
+
+interface IGenePanelModalProps {
+    genePanelCache: GenePanel;
```

Comment:
looks like its posible for genePanelCache not to exist, based on line 26. so type should be `genePanelCache?:...`. Also, why is it called `genePanelCache`, when its just a single `GenePanel` object?

Reply:
I kept the prop name from the previous version of the GenePanelModal component, will change it to genePanel.